### ISSUE TITLE
Cannot pass props into a SecureRoute in okta-react

### ISSUE THREAD

**Issue by pallavi** [okta/okta-oidc-js#78]
Hello,

I would like to pass a prop to a component that I am rendering with `SecureRoute`.

In react-router, this can be done with something like this: 
`<Route path='/path' render={() => <MyComponent prop={x}>} />`

It seems that I cannot use the render function with `SecureRoute`. Is there a way to pass props?
  
**Suggested fix:**
Change this:
https://github.com/okta/okta-oidc-js/blob/6243286b840f8812553338a622831d7bc945da91/packages/okta-react/src/SecureRoute.js#L53
to this:
```js
return <C {...renderProps} {...this.props.customProps} />; 
```
and we can pass in props like this:
`<SecureRoute path='/path' component={MyComponent} customProps={{ foo: 'bar' }} />`

**Comment by lboyette-okta** [comment]
Unfortunately, there's not currently a way to pass props to components in `SecureRoute`. [React Router v4 handles this by providing a `render` prop](https://github.com/ReactTraining/react-router/issues/1159#issuecomment-257185283), so we'll need to add the `render` prop to `SecureRoute`.

**Comment by robertjd** [comment]
Hi @pallavi , this should now be fixed in v0.0.11 - give it a try and let us know!
